tools/bowtie2.wdl

```diff
@@ -0,0 +1,446 @@
+#@ except: ContainerValue, CommentWhitespace
+
+version 1.1
+
+import "../data_structures/read_group.wdl"
+
+### Bowtie2 accepts several function parameters.
```

Comment:
Why triple pounds? I think this either belongs in the preamble, or should use single pounds.

Relevant bit already on `main` - https://github.com/stjudecloud/workflows/blob/main/tools/star.wdl#L821-L829